CHANGES_NEXT_RELEASE

```diff
@@ -1,8 +1,3 @@
-[cygnus-ngsi][Docker] Upgrade to just enable sinks and channels depending on ENV vars [#1482]
```

Comment:
CNRS seems a bit dirty... maybe it has crossed some releases.

It would be great to merge master on this PR  branch to update it.